src/plot.jl

```diff
@@ -64,7 +64,7 @@ const supportedplots = push!(collect(keys(translationdict)), :mixeddensity, :cor
         lags = 0:(maxlag === nothing ? round(Int, 10 * log10(length(range(c)))) : maxlag)
         # Chains are already appended in `c` if desired, hence we use `append_chains=false`
         ac = autocor(c; sections = nothing, lags = lags, append_chains=false)
-        ac_mat = convert(Array, ac)
+        ac_mat = stack(map(stack ∘ Base.Fix2(Iterators.drop, 1), ac))
```

Comment:
This seems a bit inefficient (particular the use of `stack(map(stack`) - maybe we should preallocate the desired array and copy directly? Possibly a utility function could be added to PosteriorStats?

Reply:
It may not be the most efficient, but compared to the actual `autocor` call, it should not be the computational bottleneck, and not using `stack` requires substantially more code.

It might make sense to add to PosteriorStats a function that converts a `SummaryStats` to a matrix, but this would effectively do the same thing as the inner `stack`. What kind of utility function do you have in mind?

Reply:
> What kind of utility function do you have in mind?

Basically a `convert`, `Array`/`Matrix` constructor`, or other function that performs the same thing (if the first two alternatives are not desirable). So that downstream packages (such as MCMCChains) don't have to deal with internals of `SummaryStats`.

Reply:
This line is not using any internals of `SummaryStats`, whose docstring explains it is an OrderedDict-like Table that can be iterated/indexed by its column names. 

I think the methods we're missing in PosteriorStats are
```julia
function Base.getindex(stats::SummaryStats, cols::Union{Colon,AbstractVector{Int},AbstractVector{Symbol}})
    cols isa Colon && return Tables.matrix(stats)
    return stack(Tables.getcolumn(stats, k) for k in cols)
end
Base.firstindex(s::SummaryStats) = 1
Base.lastindex(s::SummaryStats) = length(s)
```

Then this would be
```julia
stack(ac_i[2:end] for ac_i in ac)
```
but this is no more efficient.

Reply:
> This line is not using any internals of SummaryStats, whose docstring explains it is an OrderedDict-like Table that can be iterated/indexed by its column names.

I was referring to the `2:end` which seems a bit special. I missed that the docstring mentions that the first column is reserved for parameter names.